packages/mui-material/src/AvatarGroup/AvatarGroup.d.ts

```diff
@@ -40,6 +40,12 @@ export interface AvatarGroupProps extends StandardProps<React.HTMLAttributes<HTM
    * @default 5
    */
   max?: number;
+  /**
+   * custom renderer of extraAvatars
+   * @param {number} surplus number of extra avatars
+   * @returns {React.ReactElement} custom element to display
+   */
+  renderSurplus?: (surplus: number) => React.ReactElement;
```

Comment:
The `render...` props we have in other components all return `ReactNode`, should this be the case here as well?

Examples: 
- https://mui.com/material-ui/api/pagination/#Pagination-prop-renderItem
- https://mui.com/material-ui/api/select/#Select-prop-renderValue
- https://mui.com/material-ui/api/autocomplete/#Autocomplete-prop-renderGroup

Reply:
I think ReactNode fits much better. Commited your suggested changes